okapi/services/attrs/attribute-definitions.xml

```diff
@@ -50,9 +56,23 @@ Legend and field usage:
 - NAME      Attribute name.
             (use <groundspeak ... name="NAME" .../>)
 
-Note: in Opencaching implementations all attributes are either ON or OFF.
-Inverse meaning is handled by a different attribute.
+Notes: 
+- All OC implementations have attributes either ON or OFF.
+  Inverse meaning is handled by a different attribute.
+- GPX exports retain Groundspeak compatibility by providing a properly set `inc` flag
+  for all attributes, including OC attributes that have the inverse meaning of another.
+- each OC implementation provides two types of GPX exports: 
+  * one via OKAPI
+  * one via OC implementation's own code
+- OCDE implements internally the `inc` flag (in the database) for GS compatibility 
+  of it's own GPX exports (for the representation of the inverse meaning of an attribute):
+  YES <=> inc=1  
+  NO  <=> inc=0
+- OCPL generates the `inc` innternally in the code for GS compatibility of 
+  it's own GPX exports.
 
+Groundspeak attributes:
+---------------------------------------------------------------------------
```

Comment:
lineNumber: 75; columnNumber: 3; string "--" is not allowed in comments

Reply:
removed.